File: 02_ProgrammingFundamentals/_02_DataTypesAndVariables/_02_Exercise/_01_IntegerOperations.java
package _02_ProgrammingFundamentals._02_DataTypesAndVariables._02_Exercise;

import java.util.Scanner;

public class _01_IntegerOperations {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n1 = Integer.parseInt(scan.nextLine()), n2 = Integer.parseInt(scan.nextLine()), n3 = Integer.parseInt(scan.nextLine()), n4 = Integer.parseInt(scan.nextLine());

        // Calculate result
        int result = ((n1 + n2) / n3) * n4;

        // Print result
        System.out.println(result);

        scan.close();
    }
}

File: 24_C#Fundamentals/_02_DataTypesAndVariables/_02_Exercise/_01_IntegerOperations/_01_IntegerOperations.cs
using System;

public class _01_IntegerOperations
{
    public static void Main(string[] args)
    {
        // Read input values
        int n1 = int.Parse(Console.ReadLine()), n2 = int.Parse(Console.ReadLine()), n3 = int.Parse(Console.ReadLine()), n4 = int.Parse(Console.ReadLine());

        // Calculate result
        int result = ((n1 + n2) / n3) * n4;

        // Print result
        Console.WriteLine(result);
    }
}
